payment/schema.py

```diff
@@ -41,6 +44,15 @@ def resolve_payments(self, info, **kwargs):
         show_history = kwargs.get('show_history', False)
         if not show_history and not kwargs.get('uuid', None):
             filters += filter_validity(**kwargs)
+        # OFS-257: Create dynamic filters for the payment mutation
+        additional_filter = kwargs.get('additional_filter', None)
+        if additional_filter:
+            # send signal to append additional filter
+            signal_before_payment_query.send(
+                sender=self, additional_filter=additional_filter, filters=filters, user=info.context.user,
+            )
+            # distinct query result after filtering through payment details
```

Comment:
how do you retrive the signal result ?

Reply:
As I wrote to you few seconds ago on Skype - I will modify this to return signal result and process it.